torch/csrc/jit/operator_upgraders/upgraders.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include <c10/macros/Export.h>
+#include <iostream>
+#include <mutex>
+#include <string>
+#include <unordered_map>
+
+namespace torch {
+namespace jit {
+
+struct UpgradersMap {
+    static std::unordered_map<std::string, std::string> content;
```

Comment:
just declare those as globals.

Reply:
Yeah sounds good.

Reply:
I like Yanan's suggestion. To be concrete: declare content etc as non-static member fields of UpgradeMap, move functions into that class as methods; then declare `extern UpgradeMap gUpgraderMap;`  as global